gwbasic.py

```diff
@@ -1,144 +1,236 @@
 # class to load GW-BASIC programs
+import re
+import math
 from gwbasic_tokens import tokens
 
-
-# Parse an encoded float or double. GW-BASIC uses a strange, non-standard
-# format. See http://www.chebucto.ns.ca/~af380/GW-BASIC-tokens.html for
-# details.
-def ParseFloat(bytes):
-    exp = bytes[3] - 0x80
-    sgn = (bytes[2] & 0x80) >> 8
-    mantissa = 0.5 + 1.0 * (((bytes[2] & 0x7f) << 16) + (bytes[1] << 8) + bytes[0]) / 2**24
-    return (-1)**sgn * 2**exp * mantissa
-
-
-def ParseDouble(bytes):
-    return "<unparsed double>"
-    # return struct.unpack("!d", ''.join([chr(x) for x in bytes[8::-1]]))
-
-
-class gwbasicline:
+class gwBasicLine:
```

Comment:
I realize you're inheriting this, but in Python you generally CamelCase class names, so `GWBasicLine` and `GWBasic`.